### ISSUE TITLE
Supporting cross-chain WIFD from Hydration or AH to Moonbeam

### ISSUE THREAD

**Issue by dogwifdots** [galacticcouncil/hydration-ui#1475]
Woof to you hydrated people!

I want to make it as easy as possible for users to cross-chain WIFD between Hydration (or at least AH) to Moonbeam.

I used the following extrinsic to transfer WIFD from AH to Moonbeam:
0x1f0803010100511f0300010300366962ef6c05cbb62d35238d4cdc2d35816480ce030800000204320544000700e40b5402000002043205011f00c2d401000100000000

Can you please support me in the steps in order to make WIFD cross-chainable like PINK?

Regards and thanks in advance,
dogwifdots dev

**Comment by dogwifdots** [comment]
Woof hey @jak-pan,
I did a thing wif Moonbeam team and enabled xc tranfers for WIFD. This may help adding WIFD to your cross-chain UI for Asset Hub > moonbeam and from moonbeam > Asset Hub. The hydration > Asset Hub and back path is already open.

<img width="614" alt="image-1" src="https://github.com/user-attachments/assets/033f2552-c201-40c5-8a29-e9e55d66488f">

Many thanks in advance! I also informed the StellaSwap team to do add WIFD to their UI for the XC Transfers
